src/Interfaces/Options/Particles/IOptionsParticlesNumberDensity.ts

```diff
@@ -0,0 +1,4 @@
+export interface IOptionsParticlesNumberDensity {
+    enable: boolean;
+    value_area: number;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'value_area' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)